library/kani/src/contracts.rs

```diff
@@ -225,4 +225,27 @@
 //! Rust pointer type (`&T`, `&mut T`, `*const T` or `*mut T`). In addition `T`
 //! must implement [`Arbitrary`](super::Arbitrary). This is used to assign
 //! `kani::any()` to the location when the function is used in a `stub_verified`.
+//!
+//! ## History Variables
+//!
+//! Additionally, an ensures clause is allowed to refer to history computations
```

Comment:
"history computations" is not defined or a well known term. I would suggest 

 ```suggestion
//! Additionally, an ensures clause is allowed to refer to the state of the function arguments before function execution and perform simple computations on them
```